CDCgov/prime-reportstream
# 7931 UP supports patient having mobile and email information

This PR updates the Patient resource mapping from FHIR to HL7 v2 to support Patient having mobile and email information.

Test Steps:
1. Build this branch and run the unit and integration tests. Verify they all pass. Alternatively you can verify the Github Actions build passes. Note the HL7 to FHIR to HL7 integration test data was not changed, so the result of our HL7 to FHIR to HL7 conversion remains the same = no data changes.
2. A new sample FHIR file (sample_SR_1_20230119-0001.fhir) was added that contains a home phone, cell phone, business number, and email under Telecom. You can verify these values are being mapped correctly to PID.13 and PID.14 on the resulting HL7 file.

## Changes
- Updated FHIR to HL7 v2 mappings

## Checklist

### Testing

### Process

## Linked Issues
- Fixes #7931
